spec/features/admin/help_text_spec.rb

```diff
@@ -28,9 +32,11 @@
       click_on 'ヘルプテキストを更新'
     end
 
-    it { expect(page_title).to have_content("##{help_text.id}") }
-    it { expect(current_path).to eq admin_help_text_path(help_text) }
-    it { expect(page).to have_content(new_body) }
-    it { expect(help_text.reload.body).to eq new_body }
+    it 'should update the help text' do
+    expect(page_title).to have_content("##{help_text.id}")
```

Comment:
__[rubocop]__
* Use 2 (not 0) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)

<!-- Comment by [SideCI](https://sideci.com) -->